FreshAir/UI/RZFReleaseNoteCollectionViewCell.m

```diff
@@ -31,12 +33,29 @@
 static const NSInteger kRZFReleaseNoteTitleLabelNumberOfLines = 1;
 static const NSInteger kRZFReleaseNoteMessageLabelNumberOfLines = 3;
 
+static const CGFloat kRZFScreenshotImageViewFullScreenHorizontalPadding = 10.0f;
+static const CGFloat kRZFScreenshotImageViewFullScreenTopPadding = 10.0f;
+static const CGFloat kRZFScreenshotImageViewFullScreenAspectRatio = 355.0f/365.0f;
+
+static const CGFloat kRZFReleaseNoteTitleLabelFullScreenHorizontalPadding = 15.0f;
+static const CGFloat kRZFReleaseNoteTitleLabelFullScreenTopPadding = -12.0f;
+
+static const CGFloat kRZFReleaseNoteMessageLabelFullScreenHorizontalPadding = 15.0f;
+static const CGFloat kRZFReleaseNoteMessageLabelFullScreenTopPadding = 4.0f;
+static const CGFloat kRZFReleaseNoteMessageLabelHeight = 140.0f;
+
+static const NSInteger kRZFReleaseNoteMessageLabelFullScreenNumberOfLines = 0;
+
 @interface RZFReleaseNoteCollectionViewCell ()
 
 @property (strong, nonatomic) UIImageView *screenshotImageView;
 @property (strong, nonatomic) UIView *infoContainerView;
 @property (strong, nonatomic) UILabel *releaseNoteTitleLabel;
 @property (strong, nonatomic) UILabel *releaseNoteMessageLabel;
+@property (strong, nonatomic) UIView *notesContainer;
+@property (strong, nonatomic) NSLayoutConstraint *releaseNoteMessageHeightConstraint;
+@property (strong, nonatomic) RZFReleaseNotes *releaseNotes;
```

Comment:
View should not need to hold on to model objects. Can it just get passed around in the configure method?